contracts/payment/RefundEscrow.sol

```diff
@@ -0,0 +1,74 @@
+pragma solidity ^0.4.23;
+
+import "./ConditionalEscrow.sol";
+
+
+/**
+ * @title RefundEscrow
+ * @dev Escrow that holds funds for a beneficiary, deposited from multiple parties.
+ * The contract owner may close the deposit period, and allow for either withdrawal
+ * by the beneficiary, or refunds to the depositors.
+ */
+contract RefundEscrow is ConditionalEscrow {
+  enum State { Active, Refunding, Closed }
+
+  event Closed();
+  event RefundsEnabled();
+
+  State public state;
+  address public beneficiary;
+
+  /**
+   * @dev Constructor.
+   * @param _beneficiary The beneficiary of the deposits.
+   */
+  function initialize(address _sender, address _beneficiary) isInitializer("RefundEscrow", "1.11.0") public {
+    require(_beneficiary != address(0));
+    Ownable.initialize(_sender);
```

Comment:
`ConditionalEscrow` is the contract that ought to be initialized, not `Ownable`.

Reply:
Agree, but Solidity doesn't allow me to call ConditionalEscrow.initialize, since it is not explicitly implementing it